chrome-extension/content.js

```diff
@@ -101,23 +97,50 @@ async function createSidebar(content) {
   commentTextarea.setAttribute('placeholder', 'Note to self');
   form.appendChild(commentTextarea);
 
-  // Create select dropdown with options fetched from categories db
-  const dropdownLabel = customCreateElement('label', ['label', 'mt-2'], 'Add Clipping to a Category?');
-  dropdownLabel.htmlFor = 'categoryDropdown';
-  form.appendChild(dropdownLabel);
-  
   sidebar.appendChild(form);
 
-  const categoryDropdown = await getCategoryDropdown(currentUserId);
-  categoryDropdown.id = 'categoryDropdown';
-  form.appendChild(categoryDropdown);
+  const userEmailNote = customCreateElement('div', []);
```

Comment:
Would it make sense to call this userEmailWarning instead?

Reply:
oh well the main purpose of this is to display the person's email when signed in. We could have an error display that's a separate thing. =

Reply:
Ohhhhh yeah I see what you mean. I think that comes down to how you end up implementing your error handling

Reply:
yep!